health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CreateCampaign.js

```
@@ -116,11 +117,14 @@ const CreateCampaign = ({ hierarchyType, hierarchyData }) => {
     }));
 
     if (formData?.CampaignName && !editName) {
+      setIsValidatingName(true);
       let temp = await fetchValidCampaignName(tenantId, formData);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'temp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)